### ISSUE TITLE
bins for node v21

### ISSUE THREAD

**Issue by zdm** [uNetworking/uWebSockets.js#977]
Could you please release binaries for the node `v21`.

**Comment by uNetworkingAB** [comment]
We only support 3 latest LTS versions. So next change will be 16 becoming 22.

**Comment by zdm** [comment]
You are now not supporting `current` node version?
Oh, shit, this is bad news.

**Comment by uNetworkingAB** [comment]
I'm aware of how it works. Supporting LTS-only is in line with the corporate target audience. You'll have to wait 6 months for next LTS.

**Comment by zdm** [comment]
All you need to do is to add line to buils.c anf tun vuild action.
What the sense to make life complex for users.
This is unbelivable.

**Comment by uNetworkingAB** [comment]
Thank you for telling me how to run my own project. Do you have any realistic example of how running LTS versions of Node.js is detrimental to your mental health?

**Comment by uNetworkingAB** [comment]
Please do tell me how and why a LTS-only schedule hinders you from achieving your goals. What is it that critically must be available the very first day of a new non-LTS Node.js version?

It certainly doesn't ring a bell with me, how this could possibly hinder any real world usage outside of private individuals who just must download everything that is downloadable the second it is available, just because they have no real problem to solve other than downloading things for the sake of downloading things.

This is not how any corporation is run - no service or corporation will inexplicably fail from having to run LTS releases. Pretty much all Linux distros run older versions of software and lag behind a few months. AWS certainly won't force you to run non-LTS. We're talking a 6 month maximum delay - it's not like your computer will grow mold and stop woking if you lag behind the absolute top for a few months.

What the problem is, is that people have this NPM install mania where they must download everything that is downloadable just because it is released and is downloadable. It's a mental thing, not rooted in any real world problems to be solved.

* You do have Node.js LTS 20 support.
* You will get Node.js LTS 22 support.
* What is your problem?

**Comment by uasan** [comment]
I can use my example to explain the benefits of building for the current version of node.

You are absolutely right, in production there is always an LTS version, but locally we have features in the development stage that are planned to be released in half a year. In local development, it is useful to have the current version of the node in order to deal with new features locally, for example, in version 21 we stabilized and optimized the web streams, I would like to test them for my functionality of downloading large files, but I can’t, there is no build.

I’m almost sure that all this is not very convincing for you, I’m just writing so that you better understand why developers want the current version of the node

**Comment by uNetworkingAB** [comment]
Yes that is the case. For anyone hogging the very latest newly released version, there is a 6 month delay. I've never worked for a company, or heard of a company, where the tools in use must be upgraded to absolutely latest within 6 month from its first release (which would be a downgrade since v21.0.0 most definitely is less stable than v20.8.1)

Normal companies have, maybe 2 years delay from a new release until it is fully in use in prod.

Version 21 is not even going to be supported more than 6 months - it's just a temporary release that will be deprecated and unsupported in 6 months when 22 becomes the new (to be) LTS. So the act of porting to, and supporting this release is unnecessary extra work and maintenance.

This whole problem is a made up, nothing burger that comes from people's inability to think realistically and pragmatically about what they really need and what their long term actual goals are.

**Comment by uNetworkingAB** [comment]
I once worked for a company where the leader constantly got side tracked with all the newly released "frameworks" and constantly rewrote his app in any newly released "framework" just because it was hyped. He never finished the product and released it in time, but just constantly rewrote it in various "frameworks".

This is such a case, of creating problems that do not exist, just for the sake of having something to complain about or do. I swear 99% of all programming jobs are about introducing pointless tasks just for the sake of being able to scratch something from a backlog and keep your job. This whole industry is pointless and achieves nothing for humanity.

**Comment by uasan** [comment]
Version 21 is interesting to me in terms of assessing the progress of optimizing web streams in the node; it is not known when version 22 will appear on production, but as a developer, it’s interesting for me to know what progress there is in future versions of the node right now, I’m a developer (engineer) and not a manager, this purely scientific interest )
